### DISCUSSION TITLE
Possible Pitch adjustment that works similar to volume/rate of speech for more variations of voices.

### DISCUSSION THREAD

**Discussion by SaphiraGrace** [karashiiro/TextToTalk#198] [Ideas]
I am not entirely sure if this is possible to implement, but if we could have a pitch slider similarly to the volume and rate of speech adjustments, I could get further variation between my Amazon Polly voices. Would make for a lot more immersive gameplay when dialogue is being read :).

**Comment by canotrix** [comment]
Could make a way to base the voice off the selected in game voice for the character.  That way it would be more consistent with the other sounds they make.
